packages/cli/src/models/files/ProjectFile.ts

```diff
@@ -33,7 +33,7 @@ interface ProjectFileData {
   manifestVersion?: string;
   zosversion?: string;
   dependencies: { [name: string]: string };
-  contracts: { [alias: string]: string };
+  contracts: string[];
```

Comment:
Lossless automatic migration is not possible, but I think we should be able to migrate consolidating all aliases to the actual contract name. We have to do some research on how this will affect people and what we can do to mitigate it.